website/static/style.css

```
@@ -21,71 +22,92 @@ body {
     flex-direction: column;
 }
 
-#title, a#title:link, a#title:visited, a#title:hover, a#title:active {
+#title, a#title:link, a#title:visited, a#title, a#title:active {
     font-weight: 1000;
     font-size: x-large;
-    color: black;
+    color: white;
     text-decoration: none;
 }
-#subtitle, a#subtitle:link, a#subtitle:visited, a#subtitle:hover, a#subtitle:active {
-    color: black;
+#subtitle, a#subtitle:link, a#subtitle:visited, a#subtitle, a#subtitle:active {
+    color: white;
     text-decoration: none;
-    font-weight: 700;
+    font-weight: 300;
+}
+
+.tip {
+    font-style: italic;
+    font-size: 0.7em;
+    color: var(--light-text-color)
 }
 
 .grid-item{
     grid-row-gap: 3px;
 }
 
-.grid-item-title {
-    word-wrap: break-word;
-    font-weight: 700;
-}
 .grid-item-title > a {
+    display: inline-block;
     text-decoration: none;
+    font-weight: 400;
```

J'aimais assez quand la police était plus épaisse. Sans cela, certains critères d'accessibilité ne sont pas remplis: https://webaim.org/resources/contrastchecker/?fcolor=3B6AD9&bcolor=EEEEEE

```suggestion
    font-weight: bold;
```